ImplementationGuide/markdown/Datenobjekte/Operations.md

```diff
@@ -43,93 +45,163 @@ In diesem Fall ist auch ein Chaining auf weitere verknüpfte Akteure möglich: `
+      }
+    }
+  ],
+  "status": "proposed",
+  "start": "2022-12-10T09:00:00Z",
+  "end": "2022-12-10T11:00:00Z",
+  "slot": [
+    {
+      "reference": "ISiKSlotExample"
+    }
+  ],
+  "serviceType": [
+    {
+      "coding": [
+        {
+          "code": "124",
+          "system": "http://terminology.hl7.org/CodeSystem/service-type"
+        }
+      ]
+    }
+  ],
+  "specialty": [
+    {
+      "coding": [
         {
-            "name": "appt-resource",
-            "resource": {
-                "resourceType": "Appointment",
-                "status": "proposed",
-                "start": "2022-12-10T09:00:00Z",
-                "end": "2022-12-10T11:00:00Z",
-                "slot":  [
-                    {
-                        "reference": "ISiKSlotExample"
-                    }
-                ],
-                "specialty":  [
-                    {
-                        "coding":  [
-                            {
-                                "code": "010",
-                                "system": "urn:oid:1.2.276.0.76.5.114"
-                            }
-                        ]
-                    }
-                ],
-                "participant":  [
-                    {
-                        "actor": {
-                            "display": "Test Patient",
-                            "reference": "Patient/example"
-                        },
-                        "status": "accepted"
-                    }
-                ]
+          "code": "010",
+          "system": "urn:oid:1.2.276.0.76.5.114"
+        }
+      ]
+    }
+  ],
+  "participant": [
+    {
+      "actor": {
+        "display": "Test Patient",
+        "reference": "Patient/example"
+      },
+      "status": "accepted"
+    }
+  ],
+  "_priority": {
+    "extension": [
+      {
+        "url": "https://gematik.de/fhir/isik/v3/Terminplanung/StructureDefinition/ISiKTerminPriorityExtension",
+        "valueCodeableConcept": {
+          "coding": [
+            {
+              "code": "25876001",
+              "system": "http://snomed.info/sct"
             }
+          ]
         }
+      }
     ]
+  }
 }
 ```
 
-Antwort:
+Antwort des Termin-Repository:
 
 ```json
 {
-    "resourceType": "Bundle",
-    "type": "searchset",
-    "entry": [
+  "resourceType": "Appointment",
+  "id": "ISiKTerminExample",
+  "meta": {
+    "tag": [
+      {
+        "code": "external",
+        "system": "http://fhir.de/CodeSystem/common-meta-tag-de"
+      }
+    ],
+    "profile": [
+      "https://gematik.de/fhir/isik/v3/Terminplanung/StructureDefinition/ISiKTermin"
+    ]
+  },
+  "extension": [
+    {
+      "url": "https://gematik.de/fhir/isik/v3/Terminplanung/StructureDefinition/ISiKNachrichtExtension",
+      "valueReference": {
+        "reference": "Communication/ISiKNachrichtExample"
+      }
+    }
+  ],
+  "status": "booked",
+  "start": "2022-12-10T09:00:00Z",
+  "end": "2022-12-10T11:00:00Z",
```

Comment:
ist ein 2-Stunden-Slot realistisch? ich würde als Ende in dem Beispiel besser 2022-12-10T09:30:00Z verwenden. (es sei denn, dass derselbe Slot auch als Beispiel für die Slot-Ressource verwendet wird und da wirklich auf 2 Stunden gesetzt ist).

Reply:
in neuer Issue als TC lösen